packages/zmarkdown/config/remark.js

```diff
@@ -144,6 +144,8 @@ const remarkConfig = {
       ':colere2:': '/static/smileys/mechant.png',
       ':ninja:': '/static/smileys/ninja.png',
       'x(': '/static/smileys/pinch.png',
+      '>_<': '/static/smileys/pinch.png',
+      '&lt;_&gt;': '/static/smileys/pinch.png',
```

Comment:
Do we need this?

Reply:
As it is the current output of the editor, I thik it is useful.